pkg/webhook/instance_admission.go

```diff
@@ -111,6 +113,11 @@ func handleUpdate(ia *InstanceAdmission, req admission.Request) admission.Respon
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	// we explicitly ignore Metadata updates
+	if reflect.DeepEqual(old.Spec, new.Spec) && reflect.DeepEqual(old.Status, new.Status) {
+		return admission.Allowed("")
+	}
+
 	// fetch new OperatorVersion: we always fetch the new one, since if it's an update it's the same as the old one
```

Comment:
This comment is outdated since we fetch both versions now.

Reply:
Well - it's still valid, isn't it?